### PR TITLE
Service provider changes

### PR DESCRIPTION
**Pull request by suzyng83209**

### CODE REVIEW

File: frontend/src/components/ServiceProvider.js [hunk 1]
```diff
@@ -74,19 +86,60 @@ class ServiceProvider extends React.Component {
             name: serviceProviderData.name || 'Algoma Family Services',
```
[the remark this answers is not in the document]
Comment (suzyng83209) [reply frontend/src/components/ServiceProvider.js]: Dummy Data

File: frontend/src/components/ServiceProvider.js [hunk 2]
```diff
@@ -74,19 +86,60 @@ class ServiceProvider extends React.Component {
...
+    }
+
     handleInputChange = (key) => {
         return (e) => {
             this.setState({ [key]: e.target.value });
         }
     }
 
+    handleSave = () => {
+        fetch(`/service_providers/${this.props.serviceProviderData.id}/update`, {
```
[the remark this answers is not in the document]
Comment (suzyng83209) [reply frontend/src/components/ServiceProvider.js]: I didn't look at the endpoint carefully. Thought it was a post. @silastsui

[the remark this answers is not in the document]
Comment (silastsui) [reply frontend/src/components/ServiceProvider.js]: yeah a POST request makes sense, i'll make an update for that tn

**Comment by Ruxuan** [comment]
Resolve merge conflicts

File: backend/constants.py [hunk 3]
```diff
@@ -211,5 +211,4 @@
 
 SERVICE_PROVIDER_FIELDS = ['name', 'website', 'report_year', 'report_link', 'expenses', 'client_total', 'staff_total', 'is_bookmarked', 'notes']
 
-FIREBASE_AUTH_FILE_PATH = os.getenv("PROJECTROOT") + "/instance/c4k-dashboard-firebase-adminsdk-ypbc3-c66b8c5a1c.json"
```
Comment (Ruxuan) [inline backend/constants.py]: Does this need to be changed? Environment var `PROJECTROOT` depends on where you run your `pipenv` commands. I usually run it from the root dir but it seems you run it from the frontend directory? I prob shouldn't have used env vars for this in hindsight lol.

File: backend/server.py [hunk 4]
```diff
@@ -106,7 +109,7 @@ def verify_admin(id_token):
     args = parser.parse_args()
 
     IS_DEV = args.dev
-
+    print(constants.FIREBASE_AUTH_FILE_PATH)
```
Comment (Ruxuan) [inline backend/server.py]: We could probably leave this print statement out.

File: frontend/src/components/ServiceProvider.js [hunk 5]
```diff
@@ -102,49 +159,61 @@ class ServiceProvider extends React.Component {
...
                 <div className="serviceProvider content">
-                    <div className="contentTitle">
-                        <span>{this.state.name}</span>
-                        <BookmarkIcon
-                            isBookmarked={this.state.isBookmarked}
-                            handleBookmarkClick={this.handleBookmarkClick}
-                        />
-                    </div> 
+                    <div className="contentHeader">
+                        <div className="contentTitleContainer">
```
Comment (Ruxuan) [inline frontend/src/components/ServiceProvider.js]: Material UI has `<Table>` that may be better suited for this. I haven't looked too deeply into it so it's your call.

**Pull request merged**